packages/com.vrchat.avatars/Editor/VRCSDK/SDK3A/SDK3AImportFix.cs

```diff
@@ -8,24 +8,38 @@ namespace VRCSDK.SDK3.Editor
     public class SDK3AImportFix
     {
         private const string packageRuntimePluginsFolder = "Packages/com.vrchat.avatars/Runtime/VRCSDK/Plugins";
-        private const string SDK3_IMPORTS_FIXED = "SDK3AImportsFixed";
+        private const string legacyRuntimePluginsFolder = "Assets/VRCSDK/Plugins/";
+        private const string reloadPluginsKey = "ReloadPlugins";
         
         static SDK3AImportFix()
         {
-            // Only run once per project
-            string key = Path.Combine(Application.dataPath, SDK3_IMPORTS_FIXED);
-	        
-            if (EditorPrefs.HasKey(key))
-                return;
-
-            EditorPrefs.SetBool(key, true);
-            Run();
+            var reloadsUntilRun = SessionState.GetInt(reloadPluginsKey, 0);
```

Comment:
This is a bit hacky, but seems to help with the issue where the VRCSDK plugins don't load fully sometimes. We have to wait for a couple of assembly reloads after a failure - if we try to reload them right away, there is no effect. Still need to fix the underlying issue, but hoping this will save people from needing to manually reimport plugins in the meantime.